extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -257,7 +261,23 @@ export class AmpUserNotification extends AMP.BaseElement {
 
   /** @override */
   show() {
-    this.element.style.display = '';
+    const viewerPaddingTop = this.viewer_.getPaddingTop();
+
+    // if there is no viewer padding, then no need to do any offset
+    if (viewerPaddingTop > 0) {
+      // we need to use getComputedStyle in cases where `top` is not in pixels
+      const currentTop = parseFloat(this.win_
+          ./*REVIEW*/getComputedStyle(this.element, null)
+          .getPropertyValue('top'));
+
+      const top = Math.max(viewerPaddingTop, currentTop);
```

Comment:
@dvoytenko this is why we use getComputedStyle which will return the value in px even if it was declared in em for example (tried it a couple of times on chrome and that seems to be the behavior, although i guess i have to guard for browser support here).

on the 2nd part, you are right it should be paddingTop + top. ill update